README.md

```diff
@@ -2,62 +2,111 @@
+            - This can only execute 1 category in 1 command
+            - Refer to the folder names in `tests` directory (`tests/*`), except for `tests/page-objects`
+            - Sample parameters: `about, av, books, collection, details, donation, home, login, music, profile, search`
+            - Sample execution:
+
+                `npm run test about`
+
+                `npm run test books`
 
-    `npm run test`
+                `npm run test donation`
 
-- run all the tests in headless mode by each category and generate test report by category:
+                `npm run test profile`
 
-    `./run-tests.sh`
+        - Test execution mode:
+            - The tests will run in headless mode by __default__ which will execute the tests in terminal only
+            - Headed mode will spawn a browser window in the screen and execute the tests
+            - Accepted parameters: `--headed`
+            - Sample execution:
 
-- run all the tests in headed mode (this will load multiple browsers):
+                `npm run test -- about --headed`
 
-    `npm run test:headed`
+                `npm run test about -- --headed`
 
-- create a `.env` file by copying the contents from `.env.sample` and add the respective values you want to use for testing
+        - Test execution - [debug mode](https://playwright.dev/docs/running-tests#debug-tests-with-the-playwright-inspector):
+            - This will spawn a browser window and Playwright Inspector window in the screen which will help on inspecting the whole test execution flow step by step
+            - Accepted parameters: `--debug`
+            - Sample execution:
 
+                `npm run test -- about --debug`
 
-## Running individual tests by category (headless mode)
+                `npm run test about -- --debug`
 
-- run about tests: `npm run test:about`
-- run av tests: `npm run test:av`
-- run books tests: `npm run test:books`
-- run collection tests: `npm run test:collection`
-- run details tests: `npm run test:details`
-- run home tests: `npm run test:home`
-- run login tests: `npm run test:login`
-- run music tests: `npm run test:music`
-- run search tests: `npm run test:search`
-- run profile tests: `npm run test:profile`
+        - Test execution - [UI mode](https://playwright.dev/docs/running-tests#debug-tests-in-ui-mode):
+            - This will spawn a browser window and Playwright Inspector window in the screen which will help on inspecting the whole test execution flow step by step
+            - Accepted parameters: `--ui`
+            - Sample execution:
 
+                `npm run test -- about --ui`
 
-## Running individual tests by category (headed mode)
+        - Test execution with [Trace viewer](https://playwright.dev/docs/trace-viewer) mode:
+            - Trace viewer will record the test run which will be included in the Playwright report
+            - It is disabled by default, you can enable it with the command below
+            - Accepted parameters: `trace`
```

Comment:
SUGGESTION: Naked parameters are test types, e.g. "about". "trace" is a meta-parameter, not a test type, so it should use the command option format, e.g. "--trace".

Reply:
updated the code for consistency